src/rebuild/srv.c

```diff
@@ -903,8 +903,13 @@ ds_rebuild_admin_stop(struct ds_pool *pool, uint32_t force)
 		D_INFO(DF_RB ": NOT stopping rebuild during opc %u(%s)\n", DP_RB_RGT(rgt),
 		       rgt->rgt_opc, RB_OP_STR(rgt->rgt_opc));
 	}
-	rgt_put(rgt);
 
+	/* admin stop command does not terminate op:Fail_reclaim, but it is remembered to avoid
+	 * retrying the original op:Rebuild.
+	 */
+	if (rgt->rgt_abort || (rgt->rgt_opc == RB_OP_FAIL_RECLAIM))
+		rgt->rgt_stop_admin = 1;
```

Comment:
this setting seems duplicate with above L901, is it really needed? or can it be combined to above L901 section?